pbs_light.go

```diff
@@ -311,6 +313,7 @@ func (deps *auctionDeps) auction(w http.ResponseWriter, r *http.Request, _ httpr
 		}
 		for i, bid := range pbs_resp.Bids {
 			bid.CacheID = cobjs[i].UUID
+			bid.CacheUrl = deps.cfg.CacheUrl + "/cache?" + strings.Replace(deps.cfg.Macros, "%PBS_CACHE_UUID%", bid.CacheID, 1)
```

Comment:
Two things about this.

1. Honest question: do you know if Go's compiler optimizes inline string concatenation? I googled a bit, but couldn't find anything on it. Otherwise it'll be very inefficient, since each string concatenation makes a new string & copies the old one.

2. Could we move this into a function on the`Configuration`? Macros are the best we can do with YAML/JSON because they don't have functions... but programing languages do, and a `func getCacheUrl(uuid string) string` is much clearer once you reach the code. I think it's cleanest if the macros concept stays confined to the Config module.